SimpleCom/app.manifest

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <assemblyIdentity version="1.0.0.0" name="SimpleCom.exe"/>
+  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
+    <application>
+      <!-- Windows 10 -->
+      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" maxversiontested="10.0.19042.868" />
+      <!-- Windows 8.1 -->
+      <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}"/>
+      <!-- Windows 8 -->
+      <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}"/>
+      <!-- Windows 7 -->
+      <supportedOS Id="{35138b9a-5d96-4fbd-8e2d-a2440225f93a}"/>
+      <!-- Windows Vista -->
+      <supportedOS Id="{e2011457-1546-43c5-a5fe-008deee3d3f0}"/> 
```

Comment:
It is much better if we support Windows 8.1 or earlier, but I cannot evaluate on them before the release because all of my PCs are already Windows 10. Can you remove them?

Reply:
Without this, Windows 10 puts the program into a Vista compatibility mode, I assume the older versions of Windows will do that too.

Reply:
I mean we can remove the definitions for Vista - 8.1.
I tried it, then the OS context column is empty - it looks good.

Reply:
I mean following change:

https://github.com/YaSuenag/SimpleCom/commit/c5ad0f28b8714a879993060a49c8a64a1e44402d#diff-c7456e6c7a7a92f1246a1ac13414fcdd09c367d5d3115b804da59991246ae00e

After this change, OS context in task manager is empty. (Red rect in following screen shot - it comes from Japanese locale, sorry)

![taskmgr](https://user-images.githubusercontent.com/7421132/120071413-3a5c8200-c0ca-11eb-9354-2b1439ec32dc.png)

I guess your suggestion is this, right? If you are ok, I want to merge this. (your contribution is recorded in commit log of course.)

Reply:
Technically, it's fine for Windows 10. This change of removing the other compatibility lines will tell Windows 8.1 or Windows 7 that this program needs to be in Vista compatibility mode. The down-level versions of Windows have no need to run the program in a compatibility mode either.

I would prefer that all the compatibility lines in, as that is what is recommended by the documentation.